test/aurora.test.ts

```diff
@@ -42,6 +42,17 @@ describe('Aurora', () => {
     // });
   });
   describe('options', () => {
+    it('activityStream', () => {
```

Comment:
These aren't very descriptive. Maybe:
```suggestion
  describe('When set, option:', () => {
    it('activityStream creates an RDSActivityStream', () => {
```
Somethin' like that.

Reply:
rewording the tests to move towards Literate Testing isn't a bad idea, but it is outside the scope of this PR.